module/Photo/src/Photo/Service/Admin.php

```diff
@@ -2,7 +2,9 @@
 
 namespace Photo\Service;
 
+use _HumbugBoxa991b62ce91e\Nette\FileNotFoundException;
```

Comment:
I think this file is internal to PHPStan (or whatever program you used for the static analysis) and it should not be in the code. I tested whether or not it would cause any errors and from what I can tell it runs fine. However, `FileNotFoundException` is not used anywhere in the code and it should therefore be removed (on top of it being _weird_).

See comment from @kokx on [the closed PR here](https://github.com/GEWIS/gewisweb/pull/1095#discussion_r646050472) too.

Reply:
This is indeed obviously a mistake. I'll remove it.